x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/utils.ts

```diff
@@ -191,3 +191,5 @@ export const buildGraphFromViewModels = (
 
   return { nodes, edges };
 };
+
+export const showStackedShape = (count?: number) => !!count && count > 1;
```

Comment:
nit: this utility function is relevant to nodes only, so it worth keeping it under the same folder hierarchy